src/backend/columnar/columnar_reader.c

```diff
@@ -756,7 +756,9 @@ SnapshotMightSeeUnflushedStripes(Snapshot snapshot)
 		}
 
 		default:
+		{
 			return false;
+		}
```

Comment:
You might have a slightly different version of uncrustify. 

If it passes check-style on GitHub, it seems fine to make this change. It would be preferable to do it as a separate commit to not obscure the history.

Reply:
Agreed, reinstalled an older version to match with what everyone is using, shortly will upload the changes.